### DISCUSSION TITLE
How can I make this work with Manifest v3?

### DISCUSSION THREAD

**Discussion by thianphat** [SimplifyJobs/webpack-ext-reloader#366] [Q&A]
Hi, currently getting the error WCER-2

https://github.com/rubenspgcavalcante/webpack-extension-reloader/wiki/General-Information#WER-E2

However, in MV3, we no longer use scripts, but service worker. Is there anything I can do to fix this?

**Comment by rushilsrivastava** [comment]
This now works with MV3 using [v1.1.10](https://www.npmjs.com/package/webpack-ext-reloader/v/1.1.10) and above
